test/oai/test_client.py

```diff
@@ -139,7 +139,7 @@ def test_usage_summary():
     client.print_usage_summary()
 
     # check update
-    client._update_usage_summary(response, use_cache=True)
+    response = client.create(prompt="1+3=", model=model, cache_seed=None)
```

Comment:
The original tests the case for using cache. The change tests the case of not using cache. Is that wanted?
cc @kevin666aa